logical_router_static_route.go

```diff
@@ -157,7 +156,7 @@ func (odbi *ovndb) lrsrListImp(lr string) ([]*LogicalRouterStaticRoute, error) {
 	if !ok {
 		return nil, ErrorNotFound
 	}
-
+	var lrFound bool
```

Comment:
why we need additional flag?

Reply:
because we iterate through all the lrs and update this to found = true if lr exists in ovn.